libkineto/src/ActivityType.cpp

```diff
@@ -60,6 +62,17 @@ ActivityType toActivityType(const std::string& str) {
   throw std::invalid_argument(fmt::format("Invalid activity type: {}", str));
 }
 
+bool hasPrivateUse1Type(const std::set<ActivityType>& activityTypes) {
+  for (const auto type : activityTypes) {
```

Comment:
can you just use std::set count() method ?

Reply:
Okey, Done.